.flake8

```diff
@@ -0,0 +1,9 @@
+[flake8]
+
+# E402 module level import not at top of file
+# gi.require_version() is required before later imports
+
+# W503 line break before binary operator
+# Following the latest pep8 style Guide
+
+ignore = E402, W503
```

Comment:
Please remove the no newline at EOF, you can do so with `$ echo >> .flak8`.

Reply:
Done !